server/alertmanager_test.go

```diff
@@ -0,0 +1,66 @@
+package server
+
+import (
+	"testing"
+	"time"
+
+	"github.com/percona/pmm/api/alertmanager/amclient"
+	"github.com/percona/pmm/api/alertmanager/amclient/alert"
+	serverClient "github.com/percona/pmm/api/serverpb/json/client"
+	"github.com/percona/pmm/api/serverpb/json/client/server"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	pmmapitests "github.com/Percona-Lab/pmm-api-tests"
+)
+
+func TestAlertManager(t *testing.T) {
+	res, err := serverClient.Default.Server.GetSettings(nil)
+	require.NoError(t, err)
+	require.True(t, res.Payload.Settings.TelemetryEnabled)
+	require.NoError(t, err)
+
+	t.Run("TestEndsAtForFailedChecksAlerts", func(t *testing.T) {
+		if !pmmapitests.RunSTTTests {
+			t.Skip("Skipping STT tests until we have environment: https://jira.percona.com/browse/PMM-5106")
+		}
+
+		defer restoreSettingsDefaults(t)
+
+		// Enabling STT
+		res, err := serverClient.Default.Server.ChangeSettings(&server.ChangeSettingsParams{
+			Body: server.ChangeSettingsBody{
+				EnableStt: true,
+			},
+			Context: pmmapitests.Context,
+		})
+		require.NoError(t, err)
+		assert.True(t, res.Payload.Settings.TelemetryEnabled)
```

Comment:
Why telemetry? `SttEnabled`?

Reply:
> SttEnabled?

Because I need to run STT checks


> Why telemetry?

I thought telemetry was a prerequisite for STT to work, is it not?

Reply:
Nope, `telemetry` is completely separated feature. Seems that you should check here `res.Payload.Settings.SttEnabled`

Reply:
done